server/src/controllers/aplus.ts

```diff
@@ -224,109 +225,92 @@ export const fetchAplusGrades = async (
       String(coursePartId)
     );
 
-    // TODO: There can be multiple sources
-    const gradeSource = await AplusGradeSource.findOne({
+    const gradeSources = await AplusGradeSource.findAll({
       where: {coursePartId: coursePart.id},
-    });
+    }) as (AplusGradeSourceData & {date: Date})[];
```

Comment:
Shouldn't date be included in the `AplusGradeSourceData` type?

Reply:
If we want to show it to the user when viewing grade sources, then yes. Right now we aren't doing that, though not for any particular reason, it'd be trivial to implement and certainly does no harm.